target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -14,6 +14,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch1" \
 			"0@eth1" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
 		;;
+	mikrotik,routerboard-750-r2)
+		ucidef_set_interfaces_lan_wan "eth0.1" "eth1"
+		ucidef_add_switch "switch0" \
+			"0@eth0" "1:lan:4" "2:lan:1" "3:lan:3" "4:lan:2"
+		;;
```

Comment:
MAC assignment missing at the bottom of this file?

Reply:
I forgot that mikrotik subtarget is special and has a default case here. Never mind ...